src/app/components/LayerPicker/LayerDetails.tsx

```diff
@@ -15,7 +15,8 @@ import { Layer } from '../../../oasis-nexus/api'
 import { Link as RouterLink } from 'react-router-dom'
 import { docs } from '../../utils/externalLinks'
 import { TextList, TextListItem } from '../TextList'
-import { getLayerLabels, getNetworkIcons } from '../../utils/content'
+import { getNetworkIcons } from '../../utils/content'
+import { getLayerNames } from '../../../types/layers'
```

Comment:
Why labels are defined in types folder?

Reply:
For historical reasons, I guess? They were always there; all that happened here is that I have discovered that we had two copies of the exact same function (with the same goal and implementation), so I removed one of them.

Would you prefer keeping the other one (which used to live in /utils/content), and removing the one from the types folder?

Reply:
ah ok we can move it in the future.